Yasgui/packages/yasgui/src/Tab.ts

```diff
@@ -491,6 +509,8 @@ export class Tab extends EventEmitter {
     return {
       yasqe: {
         value: yasgui ? yasgui.config.yasqe.value : Yasgui.defaults.yasqe.value,
+        infer: true,
+        sameAs: true,
```

Comment:
Yasgui doesn't have such things: infer and sameAs. How can the client can hide them? Also now the developers-guide is obsolete because there is a mention there for these two buttons.

Reply:
Fixed. Removes default values.